File: Models/Orders/CreateOrderItem.cs
namespace Examination_WebApi.Models.Orders
{
    public class CreateOrderItem
    {
        public int OrderId { get; set; }
        public int ProductId { get; set; }
        public string ProductName { get; set; } = null!;
        public int Quantity { get; set; }
    }
}

File: webapp/src/models/create-order-item.ts
export class CreateOrderItem {
    orderId:number
    productId:number
    productName:string
    quantity:number

    constructor(orderId:number, productId:number, productName:string, quantity:number){
        this.orderId = orderId
        this.productId = productId
        this.productName = productName
        this.quantity = quantity
    }
}
